tests/scanner/audit/retention_rules_engine_test.py

```diff
@@ -243,7 +243,7 @@ def test_only_max_normal_delete(self):
         """Test that a bucket's rule can guarantee the maximum_retention if its
         action is 'Delete' and the only condition is an age(<= maximum_retention)"""
         rules_engine = get_rules_engine_with_rule(RetentionRulesEngineTest.yaml_str_only_max_retention)
-        self.assertEqual(1, len(rules_engine.rule_book.resource_rules_map))
+        self.assertTrue(1 <= len(rules_engine.rule_book.resource_rules_map))
```

Comment:
This should be a failure if we don't get the exact number of expected rules.

You can use a constant if you don't want to update several lines any time you update the test rule strings.